src/custom/hooks/useWrapCallback.ts

```diff
@@ -15,11 +15,11 @@ import { isRejectRequestProviderError } from '../utils/misc'
 import { wrapAnalytics } from 'components/analytics'
 import { useDerivedSwapInfo } from 'state/swap/hooks'
 import { useCloseModals } from 'state/application/hooks'
-import { useCurrencyBalance } from 'state/connection/hooks'
 import { useTransactionConfirmModal } from '@cow/modules/swap/hooks/useTransactionConfirmModal'
 import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
 import { formatTokenAmount } from '@cow/utils/amountFormat'
 import { useWalletInfo } from '@cow/modules/wallet'
+import useCurrencyBalance from '@cow/modules/tokens/hooks/useCurrencyBalance'
```

Comment:
Don't kill me, but can we not use `export default`?

Reply:
This part will be refactored further as per the TODO in #2302 

`useCurrencyBalance` not only will NOT export the default (which i also don't like at all to use), but also should be imported from the module entry point. Also, it won't be in the `tokens` module, I have other plans for it